### PR TITLE
alpine: debian: Use the latest resolv gem

### PR DESCRIPTION
**Pull request by cosmo0920**
This version of resolv includes the fix for CPU spike issue due to DNS
resolver.

Signed-off-by: Hiroshi Hatake <hatake@calyptia.com>

### CODE REVIEW

**Comment by ashie** [comment]
Could you add a note that it's required only for Ruby 2.6.7, 2.7.3 and 3.0.1?
Later Ruby versions will have this fix.

**Comment by cosmo0920** [comment]
I added a note for using resolv v0.2.1: https://github.com/fluent/fluentd-docker-image/pull/272/commits/9bc8022a20ebb1523e7e7b67122926ea9b7b61f7

File: Makefile [hunk 1]
```diff
@@ -13,17 +13,17 @@
...
-	v1.12/alpine:v1.12.4-1.0,v1.12-1,edge \
-	v1.12/debian:v1.12.4-debian-1.0,v1.12-debian-1,edge-debian
+	v1.12/alpine:v1.12.4-1.1,v1.12-1,edge \
+	v1.12/debian:v1.12.4-debian-1.1,v1.12-debian-1,edge-debian
 #	<Dockerfile>:<version>,<tag1>,<tag2>,...
 
 # Define images for running on ARM platforms
 ARM_IMAGES := \
-	v1.12/armhf/debian:v1.12.4-debian-armhf-1.0,v1.12-debian-armhf-2,edge-debian-armhf \
+	v1.12/armhf/debian:v1.12.4-debian-armhf-1.1,v1.12-debian-armhf-2,edge-debian-armhf \
```
Comment (ashie) [inline Makefile:22]: Note: `v1.12-debian-armhf-2` trailing `2` should be fixed in the next version (already done at #270)

**Review by ashie — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (cosmo0920) [reply Makefile:22]: :sweat_smile:

**Comment by ashie** [comment]
Hmm, I'm checking a built image but it seems that it still loads Ruby's resolv.
From `$LOADED_FEATURES`
```
"/usr/local/lib/ruby/2.6.0/resolv.rb"
```

**Pull request merged**